src/validate.rs

```diff
@@ -868,20 +868,26 @@ fn validate_repos(data: &Data, errors: &mut Vec<String>) {
         }
         for team_name in repo.access.teams.keys() {
             if !github_teams.contains(&(repo.org.clone(), team_name.clone())) {
+                let is_team_archived = data.archived_teams().any(|t| t.name() == team_name);
```

Comment:
We should start taking organizations into account :) Ideally by creating some form of `Team/RepoId`, which contains both the org and the name. To avoid naming collisions across different organizations.

Reply:
right. Can this be a separate issue?

Reply:
Sure, but I would rather not add new code that doesn't consider organizations. There's a possibly bigger problem here though, as you should check the github team name (and organization), not the `team` team name (yeah, it's confusing.. xD).

Reply:
you mean I should check the [github](https://github.com/rust-lang/team/blob/92c25e89785dbedaaa5c8ed4f164cd53b3ecbf37/rust_team_data/src/v1.rs#L26) field? https://github.com/rust-lang/team/blob/92c25e89785dbedaaa5c8ed4f164cd53b3ecbf37/rust_team_data/src/v1.rs#L49

Reply:
Yeah, because `access.teams` refers to github team names, not `team` team names xD